<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CurveDialog</class>
 <widget class="QDialog" name="CurveDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>251</width>
     <height>281</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>3</number>
   </property>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>x</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>y</string>
    </property>
   </column>
  </widget>
  <widget class="QSpinBox" name="spinBoxCount">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>40</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimum">
    <number>3</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelCount">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Кол-во точек:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPenColor">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>90</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Выбрать цвет линии</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPenWeight">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Толщина:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxPenStyle">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>71</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <item>
    <property name="text">
     <string>Без линии</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Обычный</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Пунктир</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Пунктир-точка</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Пунктир-точка-точка</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelPenStyle">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Стиль:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxPenWeight">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>113</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDraw">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>290</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Рисовать</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>290</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelError">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>253</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>189</red>
         <green>147</green>
         <blue>249</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ошибка</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelBrushStyle">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>140</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Стиль:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxBrushStyle">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>140</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <item>
    <property name="text">
     <string>Без заливки</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Обычный</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Плотность 1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Плотность 2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Плотность 3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Плотность 4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Плотность 5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Плотность 6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Плотность 7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Гор. штрих</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Вер. штрих</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Клетка</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Диаг. штрих 1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Диаг. штрих 2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Диаг. штрих 3</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButtonBrushColor">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>160</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Выбрать цвет заливки</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTool">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>391</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Инструмент</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tableWidget</tabstop>
  <tabstop>spinBoxCount</tabstop>
  <tabstop>pushButtonPenColor</tabstop>
  <tabstop>comboBoxPenStyle</tabstop>
  <tabstop>spinBoxPenWeight</tabstop>
  <tabstop>pushButtonBrushColor</tabstop>
  <tabstop>comboBoxBrushStyle</tabstop>
  <tabstop>pushButtonDraw</tabstop>
  <tabstop>pushButtonCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
